cfme/tests/services/test_myservice.py

```diff
@@ -170,6 +172,39 @@ def test_service_link(appliance, context, myservice, provider):
         assert new_view.wait_displayed()
 
 
+@pytest.mark.parametrize('context', [ViaUI])
+@pytest.mark.meta(automates=[BZ(1720338)])
+def test_retire_service_with_retired_vm(appliance, context, myservice, provider):
```

Comment:
I would use a global fixture `service_vm` which will provide you `VM` and `MyService` object. It will reduce step of instantiation.